pkg/systemd/overview-cards/realmd-operation.js

```diff
@@ -211,6 +214,12 @@ function instance(realmd, mode, realm, state) {
                         realm = realmd.proxy(REALM, path);
                         $(realm).on("changed", update);
                         realm.wait(function() {
+                            // fill in unset realm/user with autodetected values
```

Comment:
It would make sense to have the input disabled while auto-discovery is running so that we avoid stepping over inserted text from the users.

So:
* disable fields
* run autodiscovery
* insert values and remove disabled attribute